src/be/app.py

```diff
@@ -24,20 +29,52 @@ def index():
 # serve the frontend's static assets
 @app.get("/static/{path:path}")
 def static(path: str):
+    print(f"static path: {path}")
     return fa.responses.FileResponse(
         path=_fe_dir / path,
     )
 
+# I don't know why this needs to be here but it breaks without it :)
+@app.get("/turn_on")
+def hello():
+    bulb.turn_on()
+    return "Hello from the server!"
 
 @app.get("/api/v1/status")
 def status():
+    #Use the `bulb` variable to get the properties of the LED strip, such as its name, power status, brightness, and color. Return these properties as an instance of the `Status` class.
+    bulb_name_variable = bulb.get_properties()['name']
+    power_status = bulb.get_properties()['power']
+    brightness = bulb.get_properties()['bright']
+    colour = bulb.get_properties()['rgb']
+    if bulb_name_variable is None:
+        bulb_name_variable = 'Unknown'
+    print(f"Bulb name: {bulb_name_variable}, Power status: {power_status}, Brightness: {brightness}, Colour: {colour}")
     return Status(
-        bulb_name="TODO",
+        bulb_name= bulb_name_variable,
+        power_status='on',
+        brightness=0,
+        colour=(0xFF, 0xFF, 0xFF)
     )
 
 @app.post("/api/v1/turn_on")
 def turn_on():
-    # TODO turn bulb on here
-    return "Hello from the server! implement this function"
+    bulb.turn_on() 
```

Comment:
From memory the `bulb.turn_on()` function actually returns a string `"ok"` on success, we could return that value instead of `"Turned on light"` as presumably it also returns something meaningful on failure

i.e.

```python
return bulb.turn_on()
```

Same thing can be applied to the turn_off function

Reply:
Done

Reply:
doesn't look done - with this comment resolved i would expect to see the function look like

```python
def turn_on():
   return bulb.turn_on()
```